tests/tests.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import os
+
+from libbs.api import DecompilerInterface
+
```

Comment:
Somewhere up here define `GHIDRA_INSTALL_PATH` as a variable, and get it from `os.getenv("GHIDRA_INSTALL_PATH")`